src/main/java/net/geforcemods/securitycraft/blocks/reinforced/ReinforcedDoorBlock.java

```diff
@@ -240,7 +240,7 @@ else if(neighborBlock != this)
 			{
 				boolean hasActiveSCBlock = BlockUtils.hasActiveSCBlockNextTo(world, pos) || BlockUtils.hasActiveSCBlockNextTo(world, pos.up());
 
-				if((hasActiveSCBlock || neighborBlock.canProvidePower(stateAbove)) && neighborBlock != this && hasActiveSCBlock != stateAbove.get(OPEN))
+				if(neighborBlock != this && hasActiveSCBlock != stateAbove.get(OPEN))
```

Comment:
Please explain.

Reply:
After I removed this condition, the bug that I thought could be fixed in another way (https://canary.discordapp.com/channels/318542314010312715/324493105313349644/680089122857156682) was suddenly gone.

Reply:
Ok, we'll leave it in and see what implications this has. Maybe it'll work out in the end.